tests/LINEBot/EventRequestParserTest.php

```diff
@@ -643,6 +644,7 @@ public function testParseEventRequest()
             $this->assertTrue($event->isUserEvent());
             $this->assertEquals('userid', $event->getUserId());
             $this->assertEquals('userid', $event->getEventSourceId());
+            $this->assertEquals($eventArrays[0], $event->toArray());
```

Comment:
`getEvent() ` ?

Reply:
I seem to have jumped to the conclusion. Sorry. I fixed.